### DISCUSSION TITLE
Hydratation ?

### DISCUSSION THREAD

**Discussion by MGREMY** [skoropadas/ng-doc#208] [General]
Hello,

I notices that when we comment out this line in the app configuration `provideClientHydration(),`, it works perfectly fine, but when activated, it crashes the build (and serve) process
Is there a way to provide client hydratation or not ?

Thanks :D

**Comment by skoropadas** [comment]
can you show an error? because ofc it should work, ngdoc documentation also [uses](https://github.com/ng-doc/ng-doc/blob/8845b9ecc91351b8ada6c59947b6b65dcec822e7/apps/ng-doc/src/app/app.config.ts#L20) this flag and SSR

**Reply by MGREMY** [reply]
Look like it's working now...
